.travis.yml

```diff
@@ -53,6 +53,10 @@ before_script:
       popd;
     fi
   - for i in ${BACKENDS}; do CMAKE_OPTS+=( -DCabana_ENABLE_${i}=ON ); done
+  - if [[ ${COVERAGE} ]]; then
+       pip install --user coverxygen &&
+       mkdir -p $HOME/.local/bin && wget -O $HOME/.local/bin/codecov https://codecov.io/bash && chmod +x $HOME/.local/bin/codecov; 
```

Comment:
OK so here it looks like you are adding doxygen to the coverage - correct?

Reply:
`coverxygen` is the new tool. I download codecov here as it seems more like a `before_script:` step.